src/bin/vip-db.ts

```diff
@@ -11,10 +11,10 @@ import command from '../lib/cli/command';
 import { trackEvent } from '../lib/tracker';
 
 void command( { usage: 'vip db' } )
-	.command( 'phpmyadmin', 'Open PhpMyAdmin console for your application database' )
+	.command( 'phpmyadmin', 'Access a read-only phpMyAdmin console for an environment database.' )
```

Comment:
Because phpMyAdmin resources are not "always on" for any VIP application, this command may first generate the resource before providing a generate access URL to authenticate their access.

Might "Generate access" be more accurate since the VIP-CLI command is always generating the authorized URL for users to access phpMyAdmin, but the VIP-CLI command itself is never the access point?